### PR TITLE
Check services sapconf, uuidd.socket, sysstat

### PR DESCRIPTION
**Pull request by alvarocarvajald**
As requested in fate#325363, add additional checks for the services sapconf.service, uuidd.socket and sysstat.service in tests/sles4sap/sacponf.

- Related ticket: https://fate.suse.com/325363
- Needles: N/A
- Verification run: http://mango.suse.de/tests/296 (SLES4SAP 15RC2 gnome), http://mango.suse.de/tests/295 (SLES4SAP 15RC2 textmode), http://mango.suse.de/tests/297 (SLES4SAP 12SP3)

### CODE REVIEW

File: tests/sles4sap/sapconf.pm [hunk 1]
```diff
@@ -83,28 +83,36 @@ sub run_developers_tests {
     type_string "cd\n";
 }
 
+sub verify_sapconf_service {
+    my $svc  = shift;
+    my $desc = shift;
```
Comment (okurz) [inline tests/sles4sap/sapconf.pm]: It's more common in os-autoinst-distri-opensuse to use array-assignment in one line to all parameters. Would you mind changing that?

Comment (alvarocarvajald) [reply tests/sles4sap/sapconf.pm]: Sure, no problem.

File: tests/sles4sap/sapconf.pm [hunk 2]
```diff
@@ -83,28 +83,35 @@ sub run_developers_tests {
...
-    die "Command 'systemctl status tuned' output is not recognized" unless ($output =~ m|$statusregex|s);
-
     assert_script_run("rpm -q sapconf");
 
-    $output = script_output "tuned-adm active";
+    verify_sapconf_service('tuned.service',   'Dynamic System Tuning Daemon');
+    verify_sapconf_service('sapconf.service', 'sapconf');
+    verify_sapconf_service('uuidd.socket',    'UUID daemon activation socket');
+    verify_sapconf_service('sysstat.service', 'Write information about system start to sysstat log')
+      if (sle_version_at_least('15'));
```
Comment (rwx788) [inline tests/sles4sap/sapconf.pm]: As now we have nice is_sle implemenation, sle_version_at_least is deprecated. So, could you please replace it with is_sle('15+') here?

Comment (alvarocarvajald) [reply tests/sles4sap/sapconf.pm]: Sure. But will do the same in the rest of the file. That's not the only use of `sle_version_at_least` there.

**Pull request merged**